src/test/spec/runner/test_event.rs

```diff
@@ -0,0 +1,97 @@
+use crate::{
+    bson::Document,
+    test::{CommandEvent, Matchable},
+};
+use serde::Deserialize;
+
+#[derive(Debug, Deserialize, PartialEq)]
+#[serde(rename_all = "snake_case")]
+pub enum TestEvent {
+    CommandStartedEvent {
+        command_name: Option<String>,
```

Comment:
This still needs to be an option because some of the json tests don't specify a command/database name in the expected events, but they need to be serialized into `CommandStartedEvent`s.